### DISCUSSION TITLE
Custom methods to get webrtc/hls viewer count of current stream ?

### DISCUSSION THREAD

**Discussion by rahul7827** [ant-media/Ant-Media-Server#6531] [General]
Hello Team

Hope everyone is doing well these days. I am developing a wordpress plugin for live sales and is about to complete. 
I am aware that there is rest api to get the viewer count. If we don't have such JS methods to get the number of viewer count then this can be a feature request.

**Why it is required ?**

Some applications like mine does not require any rest api call. Everything is getting done by JS library itself.

* My application is having OneToMany scenario. One Publisher and many player (Not conference.)

can we have a method like -

`webRTCAdaptor.getWebRTCViewerCount()`, 
`webRTCAdaptor.getHLSViewerCount()`

![image](https://github.com/user-attachments/assets/317687b3-794d-44b6-82d1-003aec85009e)

**Comment by rahul7827** [comment]
Any workaround solution without using socket.io.
Anyone from team 📢

**Reply by yashtandon113** [reply]
Hi @rahul7827 

Thanks for raising the query here.

It was good to know that you are developing the WordPress plugin. It sounds interesting 👍 

As per your request, we have created the feature request on our end. Here is the [Github issue](https://github.com/ant-media/Ant-Media-Server/issues/6538) for the feature request.

We will discuss it internally and get back to you.

**Reply by rahul7827** [reply]
Thank you @yashtandon113 
I appreciate it.
